src/main/java/com/microsoft/graph/models/AvailabilityItem.java

```diff
@@ -39,13 +40,13 @@ public final AdditionalDataManager additionalDataManager() {
     }
 
     /**
-     * The End Time.
-     * 
+     * The End Date Time.
+     * The end time of the time slot.
      */
-    @SerializedName(value = "endTime", alternate = {"EndTime"})
+    @SerializedName(value = "endDateTime", alternate = {"EndDateTime"})
     @Expose
 	@Nullable
-    public com.microsoft.graph.core.TimeOfDay endTime;
+    public DateTimeTimeZone endDateTime;
```

Comment:
this, and the start date time would be a breaking change. @ramsessanchez can you investigate why such a change ?
@andrueastman is this something you saw on dotnet?

Reply:
created an ICM to clarify the change https://portal.microsofticm.com/imp/v3/incidents/details/325687689/home